tutorials/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Earth Engine for Public Health Land Surface Temperature Example Uganda
+description: This tutorial shows how to describe, visualize, and export Earth Engine exposure data for public health research.
+author: hiyer20
+tags: comma-separated, lowercase, list, of, related, keywords
+date_published: 2019-09-17
+---
+
+
+---
+title: Earth Engine for Public Health Land Surface Temperature in Uganda Example
+
+---
+
+[Open In Code Editor](https://code.earthengine.google.com/1a68fa019a5175a0250ecba425ae5e0d)
+
+Learning objectives:
+•	Introduce remote sensing data.
```

Comment:
Please use Markdown `*` or `-` without additional spaces for bullet points, e.g.:
```
* Introduce..
* Learn..
```